packages/connect/src/data/DataManager.ts

```diff
@@ -7,11 +7,13 @@ import messages from '@trezor/protobuf/messages.json';
 
 import { parseCoinsJson } from './coinInfo';
 import { parseFirmwareReleases } from './firmwareInfo';
-import { ConnectSettings } from '../types';
+import type { ConnectSettings, LocalFirmwares } from '../types/settings';
 import { firmwareAssets } from '../utils/assetUtils'; // Adjust the path as necessary
 
 type AssetCollection = { [key: string]: Record<string, any> };
 
+let localFirmwares: LocalFirmwares = { firmwareDir: '', firmwareList: [] };
```

Comment:
Does this have to be outside of `DataManager` class, unlike other data fields?

Reply:
Right, that makes sense, I created it as `private static`  https://github.com/trezor/trezor-suite/pull/18564/files#diff-54a3dc9015ab9933751016bc35e7987f834b59b24a8d0cc05ef51d1ddabdd1c0R20